pkg/command/lint/command_test.go

```diff
@@ -0,0 +1,45 @@
+package lint
+
+import (
+	"path/filepath"
+	"runtime"
+	"testing"
+
+	"github.com/spf13/cobra"
+
+	// Register templates
+	_ "golang.stackrox.io/kube-linter/pkg/templates/all"
+)
+
+func TestCommand_InvalidResources(t *testing.T) {
+	testDataPath := getTestDataDir()
+	tests := []struct {
+		name    string
+		cmd     *cobra.Command
+		failure bool
+		output  string
+	}{
+		{name: "InvalidPodResource", cmd: createLintCommand(testDataPath+"invalid-pod-resources.yaml", "--fail-on-invalid-resource"), failure: true},
+		{name: "InvalidPVCResource", cmd: createLintCommand(testDataPath+"invalid-pvc-resources.yaml", "--fail-on-invalid-resource"), failure: true},
```

Comment:
How about adding missing file and not a yaml test cases?

Reply:
Maybe a valid one too. We can use `pod.yaml` from homebrew https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/kube-linter.rb#L38-L58